ci/integration-tests.sh

```diff
@@ -6,22 +6,32 @@ if [[ -z "$INTEGRATION" ]]; then
 fi
 
 rm ~/.cargo/bin/cargo-clippy
-cargo install --force --debug --path .
+cargo install --force --debug --path . || exit 1
```

Comment:
Wouldn't setting `set -e` be enough?

Reply:
cargo clippy may return non zero status code. We could suppress the error code through.

Reply:
Ah ok, I think just adding `|| true` to the clippy call would be easier then.